orbit-2-coroutines/src/test/java/com/babylon/orbit2/CoroutinePluginDslBehaviourTest.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.babylon.orbit2
+
+import com.appmattus.kotlinfixture.kotlinFixture
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.flow.onEach
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+
+internal class CoroutinePluginDslBehaviourTest {
+    private val fixture = kotlinFixture()
+    private val initialState = fixture<TestState>()
+
+    companion object {
+        @JvmStatic
+        @BeforeAll
+        fun beforeAll() {
+            Orbit.registerDslPlugins(CoroutinePlugin)
```

Comment:
since there is no `afterAll` function, we are changing global state here and not resetting it

Reply:
this also highlights that there is no test for the scenario when you try to us `transformSuspend` but haven't registered the plugin

Reply:
there are tests for missing plugins in `OrbitPluginRegistryTest`